components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -274,6 +274,11 @@ export interface GitpodServer extends JsonRpcServer<GitpodClient>, AdminServer,
      * getIDToken - doesn't actually do anything, just used to authenticat/authorise
      */
     getIDToken(): Promise<void>;
+
+    /**
+     * whether this is a single-org installation
+     */
+    isSingleOrgInstallation(): Promise<boolean>;
```

Comment:
The moment we get a second such config, we should change this to `getCellFeatures` or similar to allow for extension

Reply:
I hope not, as this is no longer meant to be a "product-line" feature flag for dedicated under which we would group more stuff. I.e. it should not be used to figure out if we are on dedicated or not, but only whether we are in a single org installation. I.e. answering questions such as Can I delete the org?, do I need to setup the installation (i.e.create the single org)?.

Reply:
Yes, but say we have another config for say billing or otherwise which is `isFoo` then exposing it under `cellFeatures` which returns 
```
{
  isSingleOrgInstallation: true/false,
  isFoo: true/false
}
```
is more practical, then adding yet another RPC